.devcontainer/devcontainer.json

```diff
@@ -1,17 +1,32 @@
 // If you want to run as a non-root user in the container, see .devcontainer/docker-compose.yml.
 {
 	"name": "Go",
-	"dockerComposeFile": "docker-compose.yml",
-    "service": "docker-in-docker",
+	"dockerFile": "./Dockerfile",
 	"workspaceFolder": "/workspace",
+	"context": "..",
+	"workspaceMount": "src=${localWorkspaceFolder},dst=/workspace,type=bind",
 	"postCreateCommand": "make set-kindcluster",
-    "shutdownAction": "stopCompose",
+    "runArgs": [
+		// // Uncomment the next line to use a non-root user. On Linux, this will prevent
+		// // new files getting created as root, but you may need to update the USER_UID
+		// // and USER_GID in .devcontainer/Dockerfile to match your user if not 1000.
+		// "-u", "vscode",
+
+		"--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined",
+		// Mount docker socket for docker builds
+		"-v", "/var/run/docker.sock:/var/run/docker.sock",
+		"--network", "host",
+		"-e GO111MODULE=on",
+		// Mount azure
+		"-v", "${env:HOME}${env:USERPROFILE}/.azure:/home/vscode/.azure"
+		// Optionally mount k8s auth to existing cluster
+		// "-v", "${env:HOME}${env:USERPROFILE}/.kube:/home/vscode/.kube",
```

Comment:
Will this work for Windows users?

Reply:
Yeah based on my testing. For those on mac/linux the `$HOME` env is set and for those on Windows the `$USERPROFILE` is set so works on both platforms as the empty one is ignored. 

There is one exception, if you launch vscode from a powershell instance then it gets upset as $HOME is set in powershell.

Reply:
I switched to your fork/branch and tried it on my windows machine and I'm getting below error

```
"C:\Program Files\Docker\Docker\Resources\bin\docker.exe" run -a STDOUT -a STDERR --mount src=c:\Lab\go\src\github.com\azure-databricks-operator,dst=/workspace,type=bind -l vsch.quality=stable -l vsch.remote.devPort=0 -l vsch.local.folder=c:\Lab\go\src\github.com\azure-databricks-operator --env-file c:\Lab\go\src\github.com\azure-databricks-operator/.env -v dboperator-gomodcache:/go/pkg -v dboperator-vscodecache:/root/.vscode-server -v dboperator-bashhistory:/root/commandhistory --cap-add=SYS_PTRACE --security-opt seccomp=unconfined -v /var/run/docker.sock:/var/run/docker.sock --network host -e GO111MODULE=on -v C:\Users\Administrator/.azure:/home/vscode/.azure --entrypoint /bin/sh vsc-azure-databricks-operator-a9f46514903195eb48a44e8cdfc60067
C:\Program Files\Docker\Docker\Resources\bin\docker.exe: open c:\Lab\go\src\github.com\azure-databricks-operator/.env: The system cannot find the file specified.
See 'C:\Program Files\Docker\Docker\Resources\bin\docker.exe run --help'.
```

Reply:
to fix it need to change to `"--env-file", "${localWorkspaceFolder}/.devcontainer/.env",`

Reply:
Fix, when testing I had my `.env` file in project root - sorry.